docs/netlify.toml

```diff
@@ -1,3 +1,3 @@
 [build]
 # only build on netlify if there are any change in docs
-ignore = "git diff --quiet 'HEAD^' HEAD ."
```

Comment:
Since we extract jsdoc from `packages/rolldown`, you probably should add it too. The generate process also relies on the commands in `package.json`, you need to add them too. @IWANABETHATGUY